libs/platform/src/lib/components/form/datetime-picker/datetime-picker.component.ts

```diff
@@ -0,0 +1,345 @@
+import {
+    AfterViewInit,
+    ChangeDetectionStrategy,
+    ChangeDetectorRef,
+    Component,
+    ElementRef,
+    EventEmitter,
+    Host,
+    Input,
+    OnDestroy,
+    OnInit,
+    Optional,
+    Output,
+    Self,
+    SkipSelf,
+    ViewChild
+} from '@angular/core';
+import { NgControl, NgForm } from '@angular/forms';
+import {
+    CalendarYearGrid,
+    DatetimePickerComponent,
+    DaysOfWeek,
+    FdCalendarView,
+    FdDate,
+    FdDatetime,
+    SpecialDayRule
+} from '@fundamental-ngx/core';
+import { Placement } from 'popper.js';
+import { BaseInput } from '../base.input';
+import { FormField } from '../form-field';
+import { FormFieldControl, Status } from '../public_api';
+
+@Component({
+    selector: 'fdp-datetime-picker',
+    templateUrl: './datetime-picker.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [{ provide: FormFieldControl, useExisting: PlatformDatetimePickerComponent, multi: true }]
+})
+export class PlatformDatetimePickerComponent extends BaseInput implements OnInit, AfterViewInit, OnDestroy {
+    /**
+     * value for datetime in string
+     */
+    @Input()
+    get value(): string {
+        return super.getValue();
+    }
+    set value(value: string) {
+        super.setValue(value);
+    }
+
+    /** Setter for the _meridian property. */
+    @Input()
+    set meridian(value: boolean) {
+        this.format = this._meridian ? 'MM/dd/yyyy, h:mm:ss a' : 'MM/dd/yyyy, H:mm:ss';
```

Comment:
Probably we have to use `value ? 'MM/dd/yyyy' ...` expression, cause at this line `this._meridian` is outdated already.

Reply:
Let's remove it since this logic is presented in the core datetime picker already

Reply:
removed the getter/setter entirely as discussed